website/content/v1.12/introduction/prodnotes.md

```diff
@@ -265,7 +263,12 @@ Follow these steps to patch your machine configuration:
 
 Additionally, you can learn more about [patches]({{< relref "../talos-guides/configuration/patching/" >}}) from the configuration patches documentation.
 
-## Step 10: Apply the Machine Configuration
+## Step 10: Configure Your Multihomed Machines
+
+If your machines are multihomed, i.e., they have more than one IPv4 and/or IPv6 addresses other than loopback, then additional configuration is required.
```

Comment:
this should come before applying machine configuration (you need to patch it for multi-homing), but idk what reads better

Reply:
> this should come before applying machine configuration (you need to patch it for multi-homing), but idk what reads better

Yes this step does come before the apply machine configuration step in the docs also.